### DISCUSSION TITLE
使用 Telegram - forecho's Blog

### DISCUSSION THREAD

**Discussion by giscus** [forecho/hugo-blog#24] [Announcements]
# 使用 Telegram - forecho's Blog

引言
今天给大家分享一下 Telegram 的使用经验，以及我为什么会喜欢 Telegram。
Telegram 是什么？
Telegram 是一个跨平台的 IM 软件，需要注意的是在国内使用 Telegram 需要自备梯子。
Telegram 的特点
其实我注册 Telegram 很早了，但是最近这 2 年才开始真正的用起来。
下面我总结一下我喜欢 Telegram 的几个特点：

https://blog.forecho.com/use-telegram.html

**Comment by ZN67** [comment]
感谢分享！希望有更多给刚翻墙的中国人的软件推荐

**Comment by songshua295** [comment]


**Reply by forecho** [reply]
其实我很早就注册 Telegram 了，只是在刚开始的时候用的很少，我甚至刚注册的时候到处找群，好玩的频道，那个时候身边的人很少用，但是最近这两年发现用的人多了。

如果是新手，欢迎加我的群 https://t.me/BaoFuTogether

**Comment by gongxg0302** [comment]
请问韭菜回本群是炸了嘛？
